cupy/math/misc.py

```diff
@@ -55,8 +55,10 @@ def convolve(a, v, mode='full'):
 
 def _fft_convolve(a1, a2, mode):
 
+    offset = 0
     if a1.size < a2.size:
         a1, a2 = a2, a1
+        offset = a2.size % 2 == 0
```

Comment:
`1 - a2.size % 2` feels better because `a2.size % 2 == 0` is of boolean type.
```suggestion
        offset = 1 - a2.size % 2
```